test/test.py

```diff
@@ -269,7 +267,10 @@ def testSZ3(self):
             self.assertTrue(original.shape == compressed.shape, "Incorrect shape")
             self.assertFalse(numpy.alltrue(original == compressed),
                              "Values should not be identical")
-            self.assertTrue(numpy.allclose(compressed, compressed_back),
+            # Absolute error from L2 norm param from:
+            # https://github.com/szcompressor/SZ3/blob/v3.1.8/include/SZ3/utils/Statistic.hpp#L44
+            abs_error = numpy.sqrt(3.0/compressed.ndim) * value
```

Comment:
Really unsure about this one, but it seems to be the way l2 norm argument is converted to abs error...
I also tried to check the L2 norm of the errors but it's way  above the given value.
Anyway, I'd take any better approach,

Reply:
Indeed, I was expecting an L2 norm calculation somewhere...